### ISSUE TITLE
Dataset with more then 1000 Datafiles is not written to archive Storage

### ISSUE THREAD

**Issue by RKrahl** [icatproject/ids.server#43]
This is likely related to Issue #42, but the symptom and also the severity is different, so I guess it's better to open a separate issue.

The IDS 1.5.0 is configured with two level storage by dataset.  After uploading 1000 datafiles to a single dataset, nothing is written to the archive storage.  The DsWriter fails with an exception.  The relevant lines of ids.log:
```
2016-02-18 11:43:35,071 [FSM Timer] DEBUG FiniteStateMachine$DsProcessQueue - Will process 8081/3061 (ESNF/12100409-ST/1.1-P/test_idsHugeRequest) with WRITE_REQUESTED
2016-02-18 11:43:35,082 [Thread-57] DEBUG DsWriter - Creating /srv/ids/storage/no-icat/cache/dataset/1759777972456197214.tmp
2016-02-18 11:43:35,133 [Thread-57] ERROR DsWriter - Write of 8081/3061 (ESNF/12100409-ST/1.1-P/test_idsHugeRequest) failed due to class org.icatproject.IcatException_Exception attempt to return more than 1000 entitities
```

Then I tried to force the Tidier to archive this dataset (by uploading some more big datasets).  The DsArchiver also fails with an exception (with a funny message):
```
2016-02-18 13:13:40,170 [FSM Timer] DEBUG FiniteStateMachine$DsProcessQueue - Will process 8081/3061 (ESNF/12100409-ST/1.1-P/test_idsHugeRequest) with ARCHIVE_REQUESTED
2016-02-18 13:13:44,757 [Thread-67] ERROR DsArchiver - Archive of 8081/3061 (ESNF/12100409-ST/1.1-P/test_idsHugeRequest) failed due to /srv/ids/storage/no-icat/data/ESNF/121/12100409-ST/1.1-P/data/test_idsHugeRequest
```

Note that none of the errors are visible to the user.

**Comment by fisherab** [comment]
This has already been addressed in 1.6.0 where maxEntities is specified in the ids.properties file and where it may not exceed the value in the icat.properties file. If I place a dependency of the ids upon icat 4.6.0 then a better solution can be implemented as the icat can be queried for it's maxEntitites value. I will label this as a duplicate. If people are happy with a dependency upon icat 4.6.0 and higher then please say so.

**Comment by RKrahl** [comment]
See comment in #42.
